### PR TITLE
Redirect to login for session proposal if not logged in.

### LINKED ISSUE #3193: Add Login/SignUp for speakers if not logged in
If the user is not logged in and clicks on "Submit Proposal" below call for speakers, show a message "Please login to submit your proposal and show a screen option "Login or Signup".

### PR DESCRIPTION
**Pull request by magdalenesuo**
fixes #3193 
screenshot from 2017-02-13 04-08-49

### CODE REVIEW

File: app/templates/gentelella/guest/event/cfs.html [hunk 1]
```diff
@@ -30,16 +30,35 @@ <h1 style="font-weight: 300; font-size: 30px">{{ _("Call for Speakers") }}</h1>
...
+                    <div id="login-user-modal" class="modal fade" tabindex="-1" role="dialog">
+                        <div class="modal-dialog">
+                            <div class="modal-content">
+                                <div class="modal-header">
+                                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"></button>
+                                    <h4 class="modal-title">{{ _("Please login to submit your proposal") }} </h4>
+                                </div>
+                                <div class="modal-footer">
+                                    <a type="button" class="btn btn-default" data-dismiss="modal">{{ _("Cancel") }}</a>
+                                    <a href="{{url_for('admin.login_view', next=request.url)}}" type="button" class="btn btn-success" id = "login-signup-button">{{ _("Login/SignUp") }}</a>
```
Comment (SaptakS) [inline app/templates/gentelella/guest/event/cfs.html]: Add spaces before and after }} and {{ respectively.

File: app/templates/gentelella/guest/event/cfs.html [hunk 2]
```diff
@@ -30,16 +30,35 @@ <h1 style="font-weight: 300; font-size: 30px">{{ _("Call for Speakers") }}</h1>
...
+                    {% if current_user.is_authenticated %}
+                        <a href="/e/{{ event.identifier }}/cfs/new">
+                            <button class="btn btn-success" data-toggle="modal"
+                                data-target="#session-modal">{{ _("Submit Proposal") }}
+                            </button>
+                        </a>
+                    {% else %}
                         <button class="btn btn-success" data-toggle="modal"
-                                data-target="#session-model">{{ _("Submit Proposal") }}
+                            data-target="#login-user-modal">{{ _("Submit Proposal") }}
```
Comment (SaptakS) [inline app/templates/gentelella/guest/event/cfs.html]: Please correct the indentation.

**Comment by mariobehling** [comment]
Travis failed.

The command "nosetests tests/unittests -v --with-coverage" exited with 1.
0.16s$ psql -c 'drop database if exists test;' -U postgres
DROP DATABASE

**Comment by SaptakS** [comment]
@mariobehling, that's the timezone error. Will get fixed after restarting build. Nothing to do with the PR.

**Comment by mariobehling** [comment]
Yes, I thought so. Passing now.

**Pull request merged**